src/Fabulous.AST.Tests/WidgetTests.fs

```diff
@@ -70,6 +70,50 @@ let x = 12
         
 let x = 12
 
+"""
+
+    [<Test>]
```

Comment:
I would also bite the bullet now and use separate files for the different widgets.